charon/src/translate/translate_functions_to_ullbc.rs

```diff
@@ -1833,7 +1833,7 @@ impl<'tcx, 'ctx> TranslateCtx<'tcx, 'ctx> {
         trace!("Translating function signature");
         let signature = bt_ctx.translate_function_signature(rust_id)?;
 
-        let body = if !is_trait_method_decl {
+        let body = if !is_trait_method_decl && !item_meta.opaque {
```

Comment:
In order to keep the code consistent, this check should not be here, but in `translate_body`. To make this work we should add a reference to an ItemMeta as input to `translate_body`.

Reply:
Should we? I feel like `translate_body` should be called when we want to translate a body.